Library/Homebrew/keg.rb

```diff
@@ -148,6 +148,11 @@ def self.all
     Formula.racks.flat_map(&:subdirs).map { |d| new(d) }
   end
 
+  sig { params(kegs: T::Array[Keg]).returns(T::Array[Keg]) }
+  def self.sort(kegs)
+    kegs.sort_by { |keg| [keg.version_scheme, keg.version] }.reverse!
+  end
```

Comment:
This is not equivalent to the previous usages of `keg_array.sort_by(&:version)` since it now does a reverse sort. Are you sure that order doesn't matter in any of the places where it was replaced?

Reply:
JSON API _maybe_ matters. The rest shouldn't.